auth/api/http/domains/transport.go

```diff
@@ -4,16 +4,17 @@
 package domains
 
 import (
+	"log/slog"
+
 	"github.com/absmach/magistrala/auth"
 	"github.com/absmach/magistrala/internal/api"
 	"github.com/absmach/magistrala/internal/apiutil"
-	mglog "github.com/absmach/magistrala/logger"
 	"github.com/go-chi/chi/v5"
 	kithttp "github.com/go-kit/kit/transport/http"
 	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
 )
 
-func MakeHandler(svc auth.Service, mux *chi.Mux, logger mglog.Logger) *chi.Mux {
+func MakeHandler(svc auth.Service, mux *chi.Mux, logger slog.Logger) *chi.Mux {
```

Comment:
```suggestion
func MakeHandler(svc auth.Service, mux *chi.Mux, logger *slog.Logger) *chi.Mux {
```

Use pointers this applies in other cases too

Reply:
@Musilah Please address these.